eventmesh-connectors/eventmesh-connector-http/src/main/resources/sink-config.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+pubSubConfig:
+  meshAddress: 127.0.0.1:10000
+  subject: TopicTest
+  idc: FT
+  env: PRD
+  group: httpSink
+  appId: 5032
+  userName: httpSinkUser
+  passWord: httpPassWord
+connectorConfig:
+  connectorName: httpSink
+  host: 127.0.0.1
+  port: 8987
+  path: /test
+  ssl: false
```

Comment:
Can we support multiple callbackUrl (path)? e.g.:

```
callbackUrl:
  - https://webhook.foo.com/callback
  - http://webhook.bar.com:8080/callback
  - ...
```

Although there is some extra code to parse the protocol header, this way we can dispense with the configuration of `ssl` parameters.

Reply:
>but do we support both common HTTP and webhooks?

For the same HTTP Sink Connector, the user must specify whether to use plain HTTP or Webhook, it is not possible to use both. Therefore, the links' type under the url list will only be one of the two. This will reduce your design pressure.

Reply:
I was asking whether coomon HTTP also needs to provide the ability to send to multiple servers, rather than supporting common and webhook simultaneously, but your answer just happened to clarify that goal. I will add that functionality in the next commit.

Reply:
Resolved at https://github.com/apache/eventmesh/pull/4837/commits/fe2b7327b744fcf0f3ea3576c892c551ed1d6fb4